src/commands/games/mcquery.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-control-regex */
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
+const { Command, CommandParameters, StringParameter } = CommandStructures
+
+module.exports = class MCQuery extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'mcquery'
+    this.category = 'games'
+    this.aliases = ['minecraftquery']
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ missingError: 'commands:mcquery.noIP' })
+    )
+  }
+  async run ({ t, author, channel }, server) {
+    let address = server.split(':')
+    let port = 25565
+    if (address.length === 2) port = address[1]
+
+    let info = await this.getInfo(address[0], port)
+    let embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    if (info.online) {
+      embed.setTitle(`${address[0]}:${port}`)
+      embed.setDescription(`**${t('commands:mcquery.stats')}:** \`${t('commands:mcquery.online')}\`\n**${t('commands:mcquery.messageOfTheDay')}:** \`${info.motd}\`\n**${t('commands:mcquery.players')}**: \`${info.currentPlayers}/${info.maxPlayers}\`\n**${t('commands:mcquery.version')}**: \`${info.version}\``)
+    } else if (info.timeout) {
+      embed.setTitle(t('commands:mcquery.unknownServer'))
+      embed.setColor(Constants.ERROR_COLOR)
+    }
+    channel.send(embed).then(() => channel.stopTyping())
+  }
+  cleanMOTD (motd) {
+    let cleanMotd = motd.split('')
```

Comment:
Ummm.... You can map() the actual string instead. There's no need to split the string cause iterators already do that

However, I think it's easier to replace all these characters with replace() + some regex.
This would work: `motd.replace(/�/g, '')`

Reply:
Well, I think I forgot line 37. The provided code would only replace `�` characters but apparently this replaces  `�` and whatever character is after it...

This should work if this is the case: `motd.replace(/�./g, '')`